scripts/pylib/twister/twisterlib.py

```diff
@@ -3501,8 +3521,8 @@ def apply_filters(self, **kwargs):
                     self.fixtures
                 )
 
-                for t in ts.cases:
-                    instance.results[t] = None
+                for tc in instance.testcases:
+                    tc.status = None
```

Comment:
it seems that we don't need this any more, because this has already been done in the TestInstance initialization process